### PR TITLE
Extract actions on new issue from models to services

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: services/issues/issues.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/log"
+	api "code.gitea.io/gitea/modules/structs"
+)
+
+// NewIssue creates new issue with labels for repository.
+func NewIssue(repo *models.Repository, issue *models.Issue, labelIDs []int64, assigneeIDs []int64, uuids []string) (err error) {
```
Comment (lafriks) [inline services/issues/issues.go]:
```suggestion
func NewIssue(repo *models.Repository, issue *models.Issue, labelIDs []int64, assigneeIDs []int64, uuids []string) error {
```

File: services/issues/issues.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	if err = models.NewIssue(repo, issue, labelIDs, assigneeIDs, uuids); err != nil {
```
Comment (lafriks) [inline services/issues/issues.go]:
```suggestion
	if err := models.NewIssue(repo, issue, labelIDs, assigneeIDs, uuids); err != nil {
```

File: services/issues/issues.go [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		return
```
Comment (lafriks) [inline services/issues/issues.go]:
```suggestion
		return err
```

File: services/issues/issues.go [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	}
+
+	if err = models.NotifyWatchers(&models.Action{
```
Comment (lafriks) [inline services/issues/issues.go]:
```suggestion
	if err := models.NotifyWatchers(&models.Action{
```

File: services/issues/issues.go [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+		Content:   fmt.Sprintf("%d|%s", issue.Index, issue.Title),
+		RepoID:    repo.ID,
+		Repo:      repo,
+		IsPrivate: repo.IsPrivate,
+	}); err != nil {
+		log.Error("NotifyWatchers: %v", err)
+	}
+
+	mode, _ := models.AccessLevel(issue.Poster, issue.Repo)
+	if err = models.PrepareWebhooks(repo, models.HookEventIssues, &api.IssuePayload{
```
Comment (lafriks) [inline services/issues/issues.go]:
```suggestion
	if err := models.PrepareWebhooks(repo, models.HookEventIssues, &api.IssuePayload{
```

**Review by lafriks — CHANGES_REQUESTED** [review]

**Comment by lunny** [comment]
@lafriks done.

File: services/issues/issues.go [hunk 6]
```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package issues
```
Comment (lafriks) [inline services/issues/issues.go]: Should use same as for other services (ex. `pull`)
```suggestion
package issue
```

**Review by lafriks — CHANGES_REQUESTED** [review]

File: services/issues/issues.go (hunk 6 above)
Comment (lunny) [reply services/issues/issues.go]: @lafriks  that could be another PR since it's not related with pull requests.

**Comment by lunny** [comment]
@lafriks done.

**Review by lafriks — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

**Pull request merged**